docs/UserGuide.md

```diff
@@ -307,28 +307,28 @@ Examples:
 * `filter_add n/filter_studentGrades n/A` Sets the dashboard to display students who have an “A” grade. 
 * `Filter_add n/filter_studentScores n/>80` Sets the dashboard to show students with test scores above 80.
 
-### 📝Grading an assignment: `grade`
-Grades a student’s assignment.
+### 📝Adding a grade to an assignment: `addgrade`
+Adds a grade to a person’s assignment.
```

Comment:
I would suggest to standardise between person and student across the User Guide depending of the user stories.